constraints/miden-vm/bitwise.air

```diff
@@ -0,0 +1,73 @@
+def BitwiseAir
+
+# Enforces that column must be binary
+ev check_binary(main: [a]):
+    enf a^2 - a = 0
+
+# Returns value aggregated from limbs
+fn aggregate(limb: vector[4]) -> scalar:
+    let result = sum([2^i * a for (i, a) in (0..4, limb)])
+    result
+
+
+public_inputs:
+    stack_inputs: [16]
+
+trace_columns:
+    main: [s, a, b, a_limb[4], b_limb[4], zp, z, dummy]
+
+periodic_columns:
+    k0: [1, 0, 0, 0, 0, 0, 0, 0]
+    k1: [1, 1, 1, 1, 1, 1, 1, 0]
+
+boundary_constraints:
+    # This is a dummy trace column to satisfy requirement of at least one boundary constraint.
+    enf dummy.first = 0
+
+integrity_constraints:
+    # Enforce that selector must be binary
+    enf check_binary([s])
+
+    # Enforce that selector should stay the same throughout the cycle.
+    enf s' = s when k1
+
+    # Enforce that the input is decomposed into valid bits
+    enf check_binary([a_limb[i]]) for i in 0..4
+    enf check_binary([b_limb[i]]) for i in 0..4  
+
+    # Enforce that the value in the first row of column `a` of the current 8-row cycle should be 
+    # the aggregation of the decomposed bit columns `a_limb`.
+    let a_aggr = aggregate([a_limb])
+    enf a = a_aggr when k0
```

Comment:
nit: Maybe we can remove `a_aggr` variable. `aggregate([a_limb])` looks concise enough to me. Not a strong opinion though.